wagtailio/templates/base.html

```diff
@@ -60,92 +60,78 @@
             <!-- End Google Tag Manager -->
         {% endif %}
 
-        {% compress css %}
-            <link rel="stylesheet" type="text/x-scss" href="{% static 'css/main.scss' %}" />
-        {% endcompress %}
+        <link rel="stylesheet" type="text/css" href="{% static 'css/main.css' %}">
 
         {% block extra_css %}{% endblock %}
 
     </head>
     <body class="{% if page %}template-{{ page.get_verbose_name|slugify }}{% endif %} {% block body_class %}{% endblock %} ">
-        {% if not DEBUG %}
-            <!-- Google Tag Manager (noscript) -->
-            <noscript>
-                <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-5XDDLH"
-                height="0" width="0" style="display:none;visibility:hidden"></iframe>
-            </noscript>
-            <!-- End Google Tag Manager (noscript) -->
-        {% endif %}
-
-        {% if FB_APP_ID %}
-        <div id="fb-root"></div>
-
-        <script>
-            window.fbAsyncInit = function() {
-                FB.init({appId: {{ FB_APP_ID }}, xfbml: true, version: 'v2.1'});
-            };
-
-            (function(d, s, id){
-            var js, fjs = d.getElementsByTagName(s)[0];
-            if (d.getElementById(id)) {return;}
-            js = d.createElement(s); js.id = id;
-            js.src = "https://connect.facebook.net/en_US/sdk.js";
-            fjs.parentNode.insertBefore(js, fjs);
-            }(document, 'script', 'facebook-jssdk'));
-        </script>
-        {% endif %}
-        {% wagtail_ab_testing_script %}
+        {% include "includes/svg_sprites.html" %}
 
         {% wagtailuserbar %}
 
         <!--[if lt IE 9]>
             <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
         <![endif]-->
```

Comment:
```suggestion
```

Further up, it would also be nice to remove `<meta http-equiv="x-ua-compatible" content="ie=edge" />`.

Reply:
Removed both